### PR TITLE
MAMPGmbH.MAMP version 4.2.0.23979

### PR DESCRIPTION
**Pull request by sadmanca**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15781)

### CODE REVIEW

**Comment by KaranKad** [comment]
`PackageIdentifier` should be `MAMP.MAMP` and `PackageName` should be actual name of the application after installation.

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
+PackageVersion: 4.2.0.23979
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 4.2.0
```

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
+PackageVersion: 4.2.0.23979
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.yaml:4]:
```suggestion
PackageVersion: 4.2.0
```

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
+PackageVersion: 4.2.0.23979
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.installer.yaml:4]:
```suggestion
PackageVersion: 4.2.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.installer.yaml:3]:
```suggestion
PackageIdentifier: MAMP.MAMP
```

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: MAMP.MAMP
```

File: manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: MAMPGmbH.MAMP
```
Comment (ItzLevvie) [inline manifests/m/MAMPGmbH/MAMP/4.2.0.23979/MAMPGmbH.MAMP.yaml:3]:
```suggestion
PackageIdentifier: MAMP.MAMP
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
cc @hossainsadman: you'll also need to rename the folder name and files.
i.e. change `MAMPGmbH` to `MAMP` as well as the `.yaml` files.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**